### ISSUE TITLE
[🐞] trying to build qwik on windows throws 

### ISSUE THREAD

**Issue by shairez** [BuilderIO/qwik#2515]
### Which component is affected?

Starters / CLI

### Describe the bug

When trying to build qwik I'm getting the following error:

![image](https://user-images.githubusercontent.com/1430726/209441620-b00bb732-dfcd-4200-97a2-da7c0002f20a.png)

seems like it's trying to go to the `runtime` dir on the root of the monorepo and not inside qwik city

### Reproduction

not relevant

### Steps to reproduce

1. `pnpm install`
2. `pnpm build.platform.copy`
3. `pnpm build`

### System Info

```shell
System:
    OS: Windows 10 10.0.19044
    CPU: (8) x64 Intel(R) Core(TM) i7-6820HQ CPU @ 2.70GHz
    Memory: 12.78 GB / 31.87 GB
  Binaries:
    Node: 18.7.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.10 - C:\Program Files\nodejs\yarn.CMD
    npm: 8.15.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Chrome: 106.0.5249.121
    Edge: Spartan (44.19041.1266.0), Chromium (108.0.1462.54), ChromiumDev (Hello HiRez.io!)
    Internet Explorer: 11.0.19041.1566
  npmPackages:
    @builder.io/partytown: 0.7.3 => 0.7.3
    vite: 4.0.1 => 4.0.1
```

**Comment by shairez** [comment]
OK this happens only when using "pnpm"

If I go to `packages/qwik-city` and run `pnpm build` I get the same error (that it can't find the "runtime" folder)

But when I run `npm run build` on the same folder, it passes 🤷‍♂️

**Comment by shairez** [comment]
this is the problematic line - 
https://github.com/BuilderIO/qwik/blob/main/scripts/qwik-city.ts#L155

if I change this to 

`const result = await execa('npm', ['run', 'build'], {`

it works...

**Comment by KenAKAFrosty** [comment]
Just reporting in to say I have the same problem still (thanks to @n8sabes for pointing this issue out to me). 

I'm actually getting a Linux VM set up right now as I type this just so I can start actively contributing to the code base 😂 Any fix here would be greatly appreciated

**Comment by adamdbradley** [comment]
@shairez what was the line you changed to get it to work?

**Comment by thejackshelton** [comment]
Currently experiencing this issue as well using windows 11 and seems to be happening to all windows users

**Comment by leifermendez** [comment]
I tried this change https://github.com/BuilderIO/qwik/pull/4777/files and it works. But I'm not sure if is the best way...

**Comment by shairez** [comment]
fixed by #4777 

Thanks again @leifermendez ! 🙏
